### DISCUSSION TITLE
With Southwest announcing assigned seats how will it affect the script?

### DISCUSSION THREAD

**Discussion by stiggy87** [jdholtz/auto-southwest-check-in#286] [General]
If people haven't learned, Southwest is moving away from their boarding process to an assigned process. 

One of the reasons why I've always used this script is to try to get in the low A-group/high B-Group for boarding. With the assigned seats coming, where will this script go? 

I know it'll take some time as Southwest upgrades their fleet of planes to the new seating arrangements.

**Comment by jdholtz** [comment]
Yeah, that's a bit unfortunate. I'm not sure what the system that they are planning to move to will look like. However, the check-in part of this script will most likely become obsolete unfortunately. I'm hoping that the fare checking aspect of the script still works though. For now, I'll continue developing it as if the check-in process were to stay. Once Southwest rolls out the changes, then I'll see how to adjust the script accordingly.

**Comment by foolio** [comment]
bummer that the checkin process will change, though if we have assigned seats then there would be no need for the checkin mad scramble anyway. glad to hear that you'll try to adapt to whatever changes come our way. 

just want to say that i've found this script immensely helpful in the past year that i discovered and started using it. lots of early morning wakeups saved and lower fares found. easily one of the top 3 most useful open source software i use. i donated a little $ and will continue to while the script is useful. thank you joey!

**Reply by jdholtz** [reply]
I really appreciate your support! It’s a fun project for me and I’m glad that others are finding it useful as well.

**Reply by darryllee** [reply]
Really really bummed about the change, in large part because of how great this script has been in basically letting me (and my family) not worry about check-ins!

**Comment by texrodin** [comment]
One suggestion on how to continue this script once assigned seating becomes fully active:
If you have a credit card, auto select the extra legroom seats at checkin (or whatever your CC allows for free).  
CC = YES
TYPE = PRIORITY
then at the designated time, script could log in to southwest and autoassign the upgraded seats to the reservation if it can find seats in a row (configurable?).

But yes, this script has kicked ass and has been a fun one to use.  Thank you @jdholtz for this badass app!

**Reply by jdholtz** [reply]
That's definitely a possibility I can see this script going in. I'm not very familiar with this specific process, so I'd have to look into it more. I'm hesitant to actually take action for a user automatically, but I'm sure there will be some use cases that the script can adapt to. The fare checking will still be around, so that will stay at least.

**Reply by markdstjohn** [reply]
A big +1 to this suggestion for choosing seats 48 hours before a flight. It'd be great if the script could simply select any seats that are next to each other so that families could be guaranteed to sit next to each other. Regarding being hesitant to take action for a user automatically, I don't think this is really an issue because the user could just login to the website and change the seats if they didn't like the ones that were selected by the script.

**Comment by wrusry** [comment]
Today was check-in for my first southwest flight where there was assigned seating. My wife bought the cheapest seats where they assign the seats at checkin. So, the program was still useful to autocheckin to get a slightly earlier checkin than I would have otherwise for this early morning flight where I wouldn't have wanted to wake up early just to check in. So, I will continue to use it.

**Comment by dpbraune** [comment]
I, too, have benefited immensely from this app and your hard work is truly appreciated.
I agree with the previous posts and suggest a thorough review of the rather confusing perks SWA is offering credit card holders, A-Listers, and A-List Preferred flyers. For example:
Earlier Group boarding
Extra seat room (if available) 48 hrs before flight time
One free checked bag
...
Currently, I'm not sure how Companion passes will work in this environment as well as holding 2 SWA credit cards (personal and business).

I'm not planning to fly before June so I'm interested in  the experiences of others.

Please keep up the great work!!

**Reply by jdholtz** [reply]
Thanks! I plan on keeping the script up to date with Southwest’s systems. Unfortunately right now I haven’t had much time nor any flights to explore Southwest’s new system (which will probably change next month). In the mean time, I’m open to any observations or contributions for further discussion on how to adapt the script.
